ha/setup/post_disruptive_upgrade.py

```diff
@@ -157,7 +157,8 @@ def _unstandby_cluster() -> None:
 
 def perform_post_upgrade(ios_instances=None, s3_instances=None, do_unstandby=False, mgmt_info=None, node_count=None):
     '''Starting routine for post-upgrade process'''
-    Log.init(service_name="post_disruptive_upgrade", log_path=RA_LOG_DIR, level="INFO")
+    ConfigManager.init(log_name="post_disruptive_upgrade",
+                                        log_path=RA_LOG_DIR, level="INFO")
```

Comment:
RA_LOG_DIR is still pointing to /var/log/seagate/cortx/ha . Please check if this should be used here

Reply:
they are not related to k8s

similar comment as above

Reply:
I did not understand what you mean here.
/var/log/seagate/cortx/ha is no longer a valid path for logging. ; so why it is used ?

Reply:
I mean this file is for pcs env for old version
so log_path is not updated but changed log.init to ConfigManager.init(

Reply:
Closing the comment, since its only for replacing log.init with configManager.init  and not for log path